### ISSUE TITLE
Is the mask decoder weight inherited from the teacher models' decoder?

### ISSUE THREAD

**Issue by Vickeyhw** [xinghaochen/TinySAM#18]
If so, in the full-stage knowledge distillation, the image encoder is randomly initialized, is the mask decoder finetuned at a smaller learning rate than the light weight image encoder? Is this consistent with your implementation?

**Comment by shuh15** [comment]
Yes, the weights for mask decoder are inherited from the teacher, and we use a smaller learning rate for mask decoder compared to image encoder.
